include/GafferImageUI/ImageGadget.h

```diff
@@ -118,6 +125,21 @@ class GAFFERIMAGEUI_API ImageGadget : public GafferUI::Gadget
 		void setSoloChannel( int index );
 		int getSoloChannel() const;
 
+		void setClipping( bool clipping );
+		bool getClipping();
+
+		void setExposure( float exposure );
+		float getExposure();
+
+		void setGamma( float gamma );
+		float getGamma();
+
+		void setDisplayTransform( GafferImage::ImageProcessorPtr displayTransform );
+		GafferImage::ImageProcessorPtr getDisplayTransform();
```

Comment:
Did you mean you wanted both a const and non-const version, or is just a const one enough?

Also, it might be worth talking about the ways in which this is kinda sketchy?  For the GPU path, we just call transform() on the node, and it could actually be const.  But for the CPU path, we actually are reconnecting the inputs of the provided displayTransform node - this API doesn't make it overly obvious that passing in a displayTransform that is reused would be very bad.  I guess it's fine, because it's mostly used from ImageView which keeps it's own internal list of display transforms which isn't used for anything else?

Reply:
I think I was suggesting a const version in addition to the non-const version, but I see you've gone with const-only now, and I think that was the right choice. We don't want external code to modify the processor after setting it, and providing only const access gives us a little more protection in that respect.